metadata-ingestion/src/datahub/ingestion/api/workunit.py

```diff
@@ -86,6 +88,22 @@ def get_urn(self) -> str:
             assert self.metadata.entityUrn
             return self.metadata.entityUrn
 
+    def get_aspects_of_type(self, aspect_cls: Type[A]) -> List[A]:
```

Comment:
this should return 0 or 1 aspects right?

imo it's reasonable to throw an error on duplication in the MCE, and update the typing

Reply:
Cool makes things simpler

Reply:
Hmm actually I don't want to fully error out here, right? Doesn't seem like a serious enough issue to completely fail an ingestion run. I can log an error but can't even report it without passing in a report to the helper... but guess that's ok. I'll go with that